packages/grafana-ui/src/components/Toggletip/Toggletip.tsx

```diff
@@ -100,6 +100,7 @@ export const Toggletip = React.memo(
       },
       middleware,
       whileElementsMounted: autoUpdate,
+      strategy: 'fixed',
```

Comment:
Fuzzy memory. Does this affect somehow the fix for https://github.com/grafana/grafana/issues/78615 ?

Reply:
nope - this PR still fixes that issue

this is on the branch with the test plugin you provided in https://raintank-corp.slack.com/archives/C025WTVRBSN/p1702636168315079:

![image](https://github.com/grafana/grafana/assets/20999846/efa9d9b2-74eb-42e9-b340-25e554cdd975)

Reply:
Thanks for double checking. I somehow had in my mind that "fixed" was causing this issue but I was wrong. :)